### PR TITLE
hash_test and hash_perf_test improvements

### PR DESCRIPTION
**Pull request by byron-marohn**
Improves #234

### CODE REVIEW

File: test/testcne/hash_perf_test.c [hunk 1]
```diff
@@ -566,24 +553,25 @@ run_all_tbl_perf_tests(unsigned int with_pushes, unsigned int ext)
     cne_printf("-----------------------------------\n");
     for (with_data = 0; with_data <= 1; with_data++) {
         if (with_data)
-            cne_printf("\n Operations with 8-byte data\n");
+            cne_printf("Operations with 8-byte data\n");
```
[the remark this answers is not in the document]
Comment (byron-marohn) [reply test/testcne/hash_perf_test.c:556]: This one is part of a big table that's printed, needs to stay printf. Same with the other nearby ones. Looks weird looking at the code but not the runnable output. But yeah, not info

File: test/testcne/hash_perf_test.c [hunk 2]
```diff
@@ -329,40 +329,40 @@ timed_lookups(unsigned int with_hash, unsigned int with_data, unsigned int table
...
+                    tst_error("Key number %u was not found", j);
                     return -1;
                 }
                 expected_data = (void *)((uintptr_t)signatures[j]);
                 if (ret_data != expected_data) {
-                    cne_printf("Data returned for key number %u is %p,"
-                               " but should be %p\n",
-                               j, ret_data, expected_data);
+                    tst_error("Data returned for key number %u is %p,"
+                              " but should be %p\n",
```
[the remark this answers is not in the document]
Comment (byron-marohn) [reply test/testcne/hash_perf_test.c:358]: Nice catch!

**Comment by byron-marohn** [comment]
Ready for final re-review

**Review by eleemhui — APPROVED** [review]
Looks good

**Review by leonmatt — APPROVED** [review]
LGTM

**Comment by byron-marohn** [comment]
pulled into test_improvements branch to submit upstream

**Pull request closed without merging**